runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -174,9 +173,6 @@ public LogData(DataType type, final Object object) {
             this.type = type;
             this.data = null;
             this.payload.set(object);
```

Comment:
When this path is invoked (during a transaction write), the global address is not set, so when sync is subsequently called, it will see that the payload is already deserialized (since its in the cache), but it won't call setGlobalAddress. Is it possible that the VLO will see this as uniitalized and thefore thing its an optimistic SMR entry ?